apps/rod/sections/summary-data-sections.js

```diff
@@ -35,11 +35,18 @@ module.exports = {
     {
       step: '/about-application',
       field: 'date-of-application',
-      parse: d => d && dateFormater.format(new Date(d))
+      parse: (d, req) => {
+        const isSponsor = req.sessionModel.get('who-is-completing') === 'sponsor';
+        return !isSponsor && d ? dateFormater.format(new Date(d)) : null;
```

Comment:
Even if we select the sponsor, we still present this page and users have to enter date of application. Any specific requirement to hide this ?